tools/copy_test_case_folders.py

```diff
@@ -6,13 +6,13 @@
 import sys
 
 # importing python folders in other direcories
-sys.path.append('/foss_fim/src/gms/')
+sys.path.append('/foss_fim/src/')
 import aggregate_branch_lists as agg
```

Comment:
This file doesn't exist in `/tools`

Reply:
tools/copy_test_case_folders.py  --- I see it in the git branch via github, so not sure what's up.

Reply:
added a possible deprecation warning, but fixed up a few things in it anyways.